entrypoint.sh

```diff
@@ -12,6 +12,8 @@ if [[ -z "$1" ]]; then
   exit 1
 fi
 
-curl -H 'Content-Type: multipart/form-data' \
--X POST \
--F "file=@$1" $DISCORD_WEBHOOK
+for f in $1; do
```

Comment:
Does this code behave as intended?

Maybe↓
```suggestion
for f in $@; do
```

Reply:
That work fine and better with your sugestion, I added more tests and change the 'Send all' test because it's reach so fast the limit of discord API.